### ISSUE TITLE
`import/order`: Order members of import with multiple members

### ISSUE THREAD

**Issue by mpsijm** [benmosher/eslint-plugin-import#2085]
When [importing multiple exports from a module](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/import#import_multiple_exports_from_module), the members themselves are not (yet) ordered by this plugin.

Perhaps this doesn't fit in `import/order` (which concerns the ordering between multiple `import` lines), but would rather be a new rule (which would concern the ordering on a single `import` line, which is importing multiple members). The new rule could be named `import/order-members`, for example.

Example of code that would pass:
```js
import {a, b, c} from "test-module";
```

Example of code that would fail:
```js
import {a, c, b} from "test-module";
```

The ESLint base rule [`sort-imports`](https://eslint.org/docs/rules/sort-imports#rule-details) already does this apparently, but I'm not using that rule, since I'm using your plugin instead, which is much more configurable.

**Comment by grit96** [comment]
You could achieve this by using both rules:

```
"sort-imports": ["error", { "ignoreDeclarationSort": true }],
"import/order": ["error", { ... }],
```

**Comment by drianoaz** [comment]
duplicate of https://github.com/benmosher/eslint-plugin-import/issues/1732
